crates/bevy_render/Cargo.toml

```diff
@@ -40,6 +40,7 @@ tracing-tracy = ["dep:tracy-client"]
 ci_limits = []
 webgl = ["wgpu/webgl"]
 webgpu = ["wgpu/webgpu"]
+gles = ["wgpu/gles"]
```

Comment:
I think we also need to expose this as a feature in the main bevy `Cargo.toml` as well? Iirc that also means it needs to be added to `bevy_internal`'s `Cargo.toml`.

Reply:
Not necessarily, people can also add bevy_render to their Cargo.toml for the feature. I think we have some other features like that that are not "broadcast" to bevy, I think in bevy_math?

Reply:
Yeah that was kinda my intention to add a bit of friction.